sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -365,6 +366,7 @@ SUBROUTINE MDLFLD
                 ! - Model output DBZ is present - proceed with calc
                 DO J=JSTA,JEND
                 DO I=1,IM
+                IF(P1D(I,J)<spval.and.T1D(I,J)<spval.and.Q1D(I,J)<spval)THEN
```

Comment:
@JesseMeng-NOAA I think the check for missing values should be placed in CALMICT so the output from CALMIC, namely CUREFL will be defined values

Reply:
the calculations in CALMICT are based on P1D, T1D, Q1D, and do not involve neighbor grid points so I consider using P,T,Q, and in the future if new versions of CALMICT implemented, this screening is still valid that we don't have to change the new CALMICT. What do you think?